packages/drivers/odsp-driver/src/createFile.ts

```diff
@@ -120,6 +116,52 @@ export async function createNewFluidFile(
     return odspResolvedUrl;
 }
 
+/**
+ * If user requested creation of a sharing link along with the creation of the file by providing either
+ * createLinkType (now deprecated) or createLinkScope in the request parameters, extract and save
+ * sharing link information from the response if it is available.
+ * In case there was an error in creation of the sharing link, error is provided back in the response,
+ * and does not impact the creation of file in ODSP.
+ * @param requestedSharingLinkKind - Kind of sharing link requested to be created along with the creation of file.
+ * @param response - Response object received from the /snapshot api call
+ * @returns Sharing link information received in the response from a successful creation of a file.
+ */
+function extractShareLinkData(
+    requestedSharingLinkKind: ShareLinkTypes | ISharingLinkKind | undefined,
+    response: any = {},
```

Comment:
Correct me if I am wrong but response here is of type "ICreateFileResponse"? Is so you can remove any and use that.